src/script/engine.zig

```diff
@@ -351,6 +352,26 @@ pub const Engine = struct {
         try self.stack.pushByteArray(value);
     }
 
+    /// OP_ROLL: Pop the top stack element as N. Move the Nth stack element to the top.
+    ///
+    /// # Returns
+    /// - `EngineError`: If an error occurs during execution
+    fn opRoll(self: *Engine) !void {
```

Comment:
`fn opRoll(self: *Engine) EngineError!void`

and you don't need this comment anymore